app/src/main/java/com/example/timer/ActiveActivity.java

```diff
@@ -0,0 +1,357 @@
+package com.example.timer;
+
+import android.annotation.SuppressLint;
+import android.app.AlertDialog;
+import android.content.ComponentName;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.ServiceConnection;
+import android.media.MediaPlayer;
+import android.os.Bundle;
+import android.os.IBinder;
+import android.view.View;
+import android.view.View.OnClickListener;
+import android.view.Window;
+import android.view.WindowManager;
+import android.widget.ImageButton;
+import android.widget.ListView;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.res.ResourcesCompat;
+import androidx.lifecycle.Observer;
+import androidx.lifecycle.ViewModelProvider;
+import java.util.ArrayList;
+
+
+public class ActiveActivity extends AppCompatActivity implements OnItemClicked {
+    public final static int  COMMAND_TICK = 100;
+    public final static int  COMMAND_STOP = 200;
+    public final static int  COMMAND_CHANGE = 300;
+
+    public static final String ACTION_PREV = "previous";
+    public static final String ACTION_PAUSE = "pause";
+    public static final String ACTION_NEXT = "next";
+
+    public final static String  ARG_PHASE = "phase";
+    public final static String  ARG_COUNTER = "counter";
+    public final static String  ARG_PHASES = "phases";
+    public final static String  ARG_TIMER = "timer";
+
+    public final static String BROADCAST_ACTION = "broadcast";
+    public final static String NOTIFICATION_ACTION = "notification";
+
+    public final static String PARAM_COMMAND = "command";
+    public final static String PARAM_ACTION = "action";
+
+    Intent serviceIntent;
+    TimerService timerService;
+
+    ActiveTimerViewModel activeViewModel;
+    TimerReceiver broadcastReceiver;
+
+    SQLitePhaseRepository sqLitePhaseRepository;
```

Comment:
It's better to work with an interface instead of implementation.

```suggestion
    PhaseRepository sqLitePhaseRepository;
```